src/immutable.js

```diff
@@ -0,0 +1,5 @@
+export function removeKeysFromObject(object, keys) {
+  return Object.keys(object)
+    .filter(k => keys.indexOf(k) === -1)
+    .reduce((acc, k) => Object.assign({}, acc, { [k]: object[k] }), {})
```

Comment:
Sadly, `Object.assign` is an ES 6 feature, and I'd like to keep compatibility with every Hyperapp user if possible. That means sticking to ES 5 in order to  support IE 10 😞

Reply:
I like the idea of an immutable version, but we can probably stick with a mutating version that loops over the keys copying to a new object 😬

Reply:
sadly, we have no other choice :'(